ROADMAP.md

```diff
@@ -0,0 +1,62 @@
+# KitOps Roadmap
+
+As a public community project the KitOps roadmap is always open to feedback. If you think a feature lower in the list is more important, or feel that we've missed an important feature please:
+
+* [Open an issue](https://github.com/jozu-ai/kitops/issues) in the repo
+* Tell us in the [KitOps Discord](https://discord.gg/Tapeh8agYy) channel
+* Send us a message on [X / Twitter](https://twitter.com/Kit_Ops)
+
+### Dev Mode (in-progress)
+
+Users can run a ModelKit packaged LLM on localhost and interact with it via chat or prompt. Uses Llama CPP with a cleaned up and more user-friendly UI. Simplifies running, testing, and integratting LLMs with applications.
+
+* Design UX for chat, prompt, and parameter interactions
+* Code UI for interactions focused on desktop form factor
+* Create ModelKits for popular LLMs
+* Add `dev` command to Kit CLI on *MacOS*
+* Add `dev` command to Kit CLI on *Windows*
+* Add `dev` command to Kit CLI on *Linux*
+
+### Tutorials (in-progress)
+
+Users can learn how to use ModelKits as part of a RAG pipeline, or while fine-tuning their LLM. Helps people get the most value out of ModelKits during their LLM development work.
+
+* Tutorial: Fine-tuning LLMs using KitOps
+* Tutorial: Doing RAG with KitOps
+
+### Attestation
+
+This feature will come in two parts: build attestation and self-attestation. Build attestations will be done by KitOps itself, adding a SLSA attestation about how the ModelKit was built. ModelKit creators can add an optional attestation for specific assets in the ModelKit, or the whole ModelKit. Users can include any 3rd party attestation Verification Summary Attestation (VSA). Additional attestations could be added as predicates. Attestation would be included as a separate layer in the ModelKit.
+
+* Add attestation for KitOps ModelKit builds
+* Add mechanism for adding attestations to ModelKits
+* CLI `pack` with attestation using a `--verify` flag
```

Comment:
This is too specific for roadmap. We have not yet decided on the details at this level.